public/js/render.js

```diff
@@ -14,9 +16,21 @@ var renderModule = function (error, eventsArray) {
     eventsSection.innerHTML = error;
   }
 
-  var filterEvents = eventsArray.filter(function (event) {
-    return event.en;
-  });
+  var filterEvents = [];
+
+  if (counter === 0) {
```

Comment:
Rather than using a counter, just check the page url, and if it's home then run the first filter